core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -1,8 +1,70 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
+import { DotListingDataTableComponent } from '@components/dot-listing-data-table/dot-listing-data-table.component';
+import { DotContainer } from '@models/container/dot-container.model';
+import { DotContentState } from '@dotcms/dotcms-models';
+import { DotContainerListStore } from '@portlets/dot-containers/container-list/store/dot-container-list.store';
+import { DotActionMenuItem } from '@models/dot-action-menu/dot-action-menu-item.model';
 
 @Component({
     selector: 'dot-container-list',
     templateUrl: './container-list.component.html',
-    styleUrls: ['./container-list.component.scss']
+    styleUrls: ['./container-list.component.scss'],
+    providers: [DotContainerListStore]
 })
-export class ContainerListComponent {}
+export class ContainerListComponent {
```

Comment:
this files and all the files in the **container-list** directory needs to have the prefix **"dot-"**

ex: DotContainerListComponent